var/spack/repos/builtin/packages/py-neurodamus/package.py

```diff
@@ -39,6 +28,8 @@ class PyNeurodamus(PythonPackage):
     depends_on('py-h5py',          type='run')
     depends_on('py-numpy',         type='run')
     depends_on('py-docopt',        type='run')
+    depends_on('py-scipy',         type='run', when='@2.5.3:')
+    depends_on('py-libsonata',     type='run', when='@2.5.3:')
```

Comment:
Same here? python interface now used directly?

Reply:
In some use cases, like ngv or nodesets, yes, we started using libsonata directly. A=But again, it's an optional dependency, i.e. neurodamus runs fine without it, until the point it absolutely requires (late import)

Reply:
We are registering dependency here so that the main deployed neurodamus works for all workloads.

Reply:
yeah, its fine. I worry about about the dependencies when people build stuff locally and then all world starts building :)

Reply:
True it could be an optional dependency also here 🤔 but then in our deployment script we would have to enable the full-dependency variant